pkg/services/datasources/service/datasource_service.go

```diff
@@ -248,26 +269,12 @@ func (s *Service) GetTLSConfig(ds *models.DataSource, httpClientProvider httpcli
 	}
 	return httpClientProvider.GetTLSConfig(*opts)
 }
-
 func (s *Service) DecryptedValues(ds *models.DataSource) map[string]string {
-	s.dsDecryptionCache.Lock()
-	defer s.dsDecryptionCache.Unlock()
-
-	if item, present := s.dsDecryptionCache.cache[ds.Id]; present && ds.Updated.Equal(item.updated) {
-		return item.json
-	}
-
-	json, err := s.SecretsService.DecryptJsonData(context.Background(), ds.SecureJsonData)
-	if err != nil {
-		return map[string]string{}
+	decryptedValues := make(map[string]string)
+	for k, v := range ds.SecureJsonData {
+		decryptedValues[k] = string(v)
 	}
-
-	s.dsDecryptionCache.cache[ds.Id] = cachedDecryptedJSON{
-		updated: ds.Updated,
-		json:    json,
-	}
-
-	return json
+	return decryptedValues
```

Comment:
Having decrypted values cached is important for datasource query performance. By removing this decrypted secrets will always be fetched when "getting" a datasource from the database. Is the idea that the secrets store will cache things instead?

Reply:
The secrets store needs to implement caching since some secret stores like AWS Secrets Manager will need to retreive secrets every time due to rotation once the interface is extendable by the SDK.

I will implement the cache on the SQL secret store 👍

Reply:
Done on 5d6985